src/library/dlgtrackinfo.h

```diff
@@ -59,6 +59,7 @@ class DlgTrackInfo : public QDialog, public Ui::DlgTrackInfo {
     void slotBpmFourThirds();
     void slotBpmThreeHalves();
     void slotBpmClear();
+    void slotBpmRound();
```

Comment:
Please also rename those slots accordingly, i.e. slotBpmRoundToInteger()

Reply:
Same applies here, rounding only means to the next integer in constant bpm mode. The new beats class might might round segments.